docs/concepts/configuration/assign-pod-node.md

```diff
@@ -253,9 +253,24 @@ A toleration "matches" a taint if the `key`s are the same and the `effect`s are
 * the `operator` is `Exists` (in which case no `value` should be specified), or
 * the `operator` is `Equal` and the `value`s are equal
 
-(`Operator` defaults to `Equal` if not specified.)
-As a special case, an empty `key` with operator `Exists` matches all keys and all values.
-Also as a special case, empty `effect` matches all effects.
+`Operator` defaults to `Equal` if not specified.
+
+**NOTE:** There are two special cases:
+
+* an empty `key` with operator `Exists` matches all keys and all values.
+
+```yaml
+tolerations:
+- operator: "Exists"
```

Comment:
and this will match all effects too, so please mention this explicity that this is tolerate everything.